presto-product-tests/src/main/java/com/facebook/presto/tests/cassandra/Select.java

```diff
@@ -232,4 +239,42 @@ public void testNationJoinRegion()
 
         assertThat(queryResult).containsOnly(row("CANADA", "AMERICA"));
     }
+
+    @Test(groups = CASSANDRA)
+    public void testMateliarizedViewSelect()
+            throws SQLException
+    {
+        String mviewName = format("%s_mview", CASSANDRA_ALL_TYPES.getName());
+        queryExecutor.executeQuery(format("DROP MATERIALIZED VIEW IF EXISTS %s.%s", KEY_SPACE, mviewName));
+        queryExecutor.executeQuery(format("CREATE MATERIALIZED VIEW %s.%s AS SELECT * FROM %s.%s WHERE b IS NOT NULL PRIMARY KEY (a, b)",
+                KEY_SPACE,
+                mviewName,
+                KEY_SPACE,
+                CASSANDRA_ALL_TYPES.getName()));
+
+        assertContainsEventually(() -> query(format("SHOW TABLES FROM %s.%s", CONNECTOR_NAME, KEY_SPACE)),
+                query(format("SELECT '%s'", mviewName)),
+                new Duration(1, MINUTES));
+
+        QueryResult query = onPresto().executeQuery(format(
+                "SELECT a, b, bl, bo, d, do, f, fr, i, integer, l, m, s, t, ti, tu, u, v, vari FROM %s.%s.%s",
+                CONNECTOR_NAME, KEY_SPACE, mviewName));
+
+        assertThat(query)
+                .hasColumns(LONGNVARCHAR, BIGINT, LONGVARBINARY, BOOLEAN, DOUBLE, DOUBLE, REAL, LONGNVARCHAR, LONGNVARCHAR,
+                        INTEGER, LONGNVARCHAR, LONGNVARCHAR, LONGNVARCHAR, LONGNVARCHAR, TIMESTAMP, LONGNVARCHAR, LONGNVARCHAR,
+                        LONGNVARCHAR, LONGNVARCHAR)
+                .containsOnly(
+                        row("\0", Long.MIN_VALUE, Bytes.fromHexString("0x00").array(), false, 0f, Double.MIN_VALUE,
+                                Float.MIN_VALUE, "[0]", "0.0.0.0", Integer.MIN_VALUE, "[0]", "{\"\\u0000\":-2147483648,\"a\":0}",
+                                "[0]", "\0", Timestamp.valueOf("1970-01-01 00:00:00.0"),
+                                "d2177dd0-eaa2-11de-a572-001b779c76e3", "01234567-0123-0123-0123-0123456789ab",
+                                "\0", String.valueOf(Long.MIN_VALUE)),
+                        row("the quick brown fox jumped over the lazy dog", 9223372036854775807L, "01234".getBytes(),
+                                true, new Double("99999999999999999999999999999999999999"), Double.MAX_VALUE,
+                                Float.MAX_VALUE, "[4,5,6,7]", "255.255.255.255", Integer.MAX_VALUE, "[4,5,6]",
+                                "{\"a\":1,\"b\":2}", "[4,5,6]", "this is a text value", Timestamp.valueOf("9999-12-31 23:59:59"),
+                                "d2177dd0-eaa2-11de-a572-001b779c76e3", "01234567-0123-0123-0123-0123456789ab",
+                                "abc", String.valueOf(Long.MAX_VALUE)));
+    }
```

Comment:
Can you also add tests for
* Selecting by partition
* more complicated select
* INSERT, if it's supported -- if not, we probably want to explicitly disallow insert in Presto with materialized views